requirements-test.txt

```diff
@@ -1,4 +1,4 @@
-ansible
+ansible-core<2.17
```

Comment:
Could you add an inline comment why we fix the version?

Reply:
Done